src/org/labkey/test/tests/SecurityTest.java

```diff
@@ -223,7 +224,10 @@ public void assertUrlForbidden(String url)
     @LogMethod
     public void passwordResetTest()
     {
-        ConfigureDbLoginPage.beginAt(this).setDbLoginConfig(PasswordStrength.Weak, null);
+        LoginConfigurePage.beginAt(this)
+                .getPrimaryConfigurationRow("Standard database authentication")
+                .clickEdit(new DatabaseAuthenticationProvider())
+                .setDbLoginConfig(DatabaseAuthConfigureDialog.PasswordStrength.Weak.Weak, null);
```

Comment:
Didn't even know you could do that.
```suggestion
                .setDbLoginConfig(DatabaseAuthConfigureDialog.PasswordStrength.Weak, null);
```